spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/AetherStubDownloaderBuilder.java

```diff
@@ -0,0 +1,21 @@
+package org.springframework.cloud.contract.stubrunner;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.cloud.contract.stubrunner.spring.StubRunnerProperties;
+
+/**
+ * @author Marcin Grzejszczak
+ * @since 2.0.0
+ */
+public class AetherStubDownloaderBuilder implements StubDownloaderBuilder {
+	private static final Log log = LogFactory.getLog(AetherStubDownloaderBuilder.class);
+
+	@Override public StubDownloader build(StubRunnerOptions stubRunnerOptions) {
+		if (stubRunnerOptions.stubsMode == StubRunnerProperties.StubsMode.CLASSPATH) {
+			return null;
+		}
+		log.info("Will download stubs from classpath");
```

Comment:
Wrong log

Reply:
😱😱😱 great catch!

Reply:
I expect mention in the release notes about this great find! :laughing: